index.html

```diff
@@ -2,84 +2,181 @@
+                        <li class="li_first text_body_font"><a href="#hello">Hello</a></li>
+                        <li class="text_body_font"><a href="#portfolio">Portfolio</a></li>
+                        <li class="text_body_font"><a href="#about">About</a></li>
+                        <li class="text_body_font"><a href="#contact">Contact</a></li>
+                    </ul>
+
+                    
+                        
+                        
+                    
+                    
+                </nav>
         </header>
 
+        
         <div class="grid_container_work">
-            <section id="hello" class="grid_item_work">
-                <div class="section_div">
-                    <div class="section_div_1">
-                        <div class="text_class text_color text_header_font">
-                            <h1>Emmanuel Mensah</h1>
-                        </div>
-                        <div class="text_subject text_color text_body_font">
-                            <p>
-                                Hello!I am a software developer! I can help you build a
-                                product, feature or website. Take a look of my works. If you
-                                like what you see and have a project you need coded, don't
-                                hesitate and contact me.
-                            </p>
-                        </div>
-                        <div class="text_button text_body_font">
-                            <p>Start Collaboration</p>
-                        </div>
+        <section id="hello" class="grid_item_work">
+            <div class="section_div">
+                <div class="section_div_1">
+                    <div class="text_class text_color text_header_font">
+                        <h1>Emmanuel Mensah</h1>
+                    </div>
+                    <div class="text_subject text_color text_body_font">
+                        <p>
+                            Hello!I am a software developer! I can
+                            help you build a product, feature or website.
+                            Take a look of my works. If you like what you see
+                            and have a project you need coded, don't hesitate
+                            and contact me.
+                        </p>
                     </div>
-                    <div class="backgroundimage"></div>
+                    <div class="text_button text_body_font">
+                        <p>Start Collaboration</p>
+                    </div>
+
                 </div>
+                <div class="backgroundimage"> </div>
 
-                <div class="mobile_image"></div>
-            </section>
+            </div>
+
+            <div class="mobile_image"></div>
+        </section>
 
+        
             <div class="grid_item_work grid_2_item">
                 <!-- work section  -->
                 <section id="portfolio" class="works_section">

```

Comment:
- [ ] [REQUIRED] As seen in the project requirement. I kindly suggest that you create a project's `array of objects` to store your project's information and create the projects dynamically in JS. Modal popup data will also be populated with the data from the projects array.

### EXAMPLE
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

Reply:
@topeogunleye Thank you for the feedback. i will implement the above issues.

Reply:
You're welcome